sites/hashdev/src/_pages/blog/10_error-stack-update-0-2.mdx

```diff
@@ -0,0 +1,185 @@
+---
+authorPhoto: blog/authors/tim-diekmann.jpg
+postPhoto: blog/10_error-stack-update-0-2/Cover.jpg
+title: "Announcing error-stack v0.2"
+subtitle: "Redesigned and customizable outputs, related errors and multiple sources, and much more"
+author: "Tim Diekmann"
+jobTitle: "Platform Engineer, HASH"
+date: "2022-10-03"
+---
+
+Back in June [we announced the first version](https://hash.dev/blog/announcing-error-stack) of `error-stack`, our context-aware error library. We developed this to help address a lot of challenges we encountered when implementing error handling in our simulation engine [LINK]. In the time since then we’ve started using it in our entity-graph query layer [LINK?] that we’re building as a datastore for HASH [LINK?]. Across that development we’ve learned a lot of lessons, and in joint efforts with open-source contributors we’re ready to release a new version of the library.
```

Comment:
Optional trimming of words
```suggestion
Back in June [we announced the first version](https://hash.dev/blog/announcing-error-stack) of `error-stack`, our context-aware error library. We developed this to help address challenges we encountered when implementing error handling in our simulation engine [LINK]. Since then we’ve started using it in our entity-graph query layer [LINK?] that we’re building as a datastore for HASH [LINK?]. Across that development we’ve learned a lot of lessons, and in joint efforts with open-source contributors we’re ready to release a new version of the library.
```

Reply:
Re-captured as a suggestion as part of https://github.com/hashintel/hash/pull/1146/files#r985691554 so marking this suggest as resolved.